src/routes/common/ActorRedirect.tsx

```diff
@@ -42,3 +42,12 @@ function ActorRedirectEnabled({ children }: ActorRedirectProps) {
     />
   );
 }
+
+function isPotentialActor(host: string) {
```

Comment:
@aashu16 see this function - we can add additional hostnames here if needed.

Right now, should support:

- Any hostname with dot (domain) or colon (port number)
- localhost

Reply:
Not sure if there is any point in being stricter with these checks.

Marking as resolved because this works for now.